localstack/services/dynamodb/provider.py

```diff
@@ -403,20 +404,21 @@ def create_table(
         if table_definitions:
             table_content = result.get("Table", {})
             table_content.update(table_definitions)
-            result["TableDescription"].update(table_content)
+            table_description.update(table_content)
 
         if "StreamSpecification" in table_definitions:
-            create_dynamodb_stream(
-                table_definitions, result["TableDescription"].get("LatestStreamLabel")
-            )
+            create_dynamodb_stream(table_definitions, table_description.get("LatestStreamLabel"))
 
         tags = table_definitions.pop("Tags", [])
-        result["TableDescription"].pop("Tags", None)
         if tags:
-            table_arn = result["TableDescription"]["TableArn"]
+            table_arn = table_description["TableArn"]
             table_arn = self.fix_table_arn(table_arn)
             DynamoDBRegion.TABLE_TAGS[table_arn] = {tag["Key"]: tag["Value"] for tag in tags}
 
+        # remove invalid attributes from result
+        table_description.pop("Tags", None)
+        table_description.pop("BillingMode", None)
```

Comment:
Minor fix for ASF response serializer (removing superfluous response attribute), unrelated to the other changes in this PR